### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by abysalvinthomas** [apohl79/audiogridder#266] [Ideas]
1 Don’t need server connection facility in each plugins, just need a common server connection window in desktop tray.
2 just need to show AG plugin and instruments under AG menu in plugin insert
3 Preset saving feature 
4 server launch ability from plugin side ( from common control panel of DAW side ). It help headless server users .

**Comment by CaBuffalo** [comment]
That'll be nice if there is a search feature so we can find the plugins quicker and easier.

**Comment by apohl79** [comment]
> 1 Don’t need server connection facility in each plugins, just need a common server connection window in desktop tray.

I see where you are coming from. I was thinking about making this connection thing easier. But you might want to connect multiple servers at the same time. Both should be possible.

> 2 just need to show AG plugin and instruments under AG menu in plugin insert

I don't understand this one. Are you referring to the midi fx version? Could you just deactivate it in the DAW plugin manager?

> 3 Preset saving feature

yeah, on the list :-)

> 4 server launch ability from plugin side ( from common control panel of DAW side ). It help headless server users .

I think plugins can't add common DAW settings, and each DAW is different. Maybe the tray icon is the way to go. But in general, the AG client needs a server to talk to. So if there is no server, it can't start one, as there is no remote end. Only restarting works and is already available.

For headless, I'm auto-starting AG server when the machine boots.

**Comment by abysalvinthomas** [comment]
Thank you for your reply.
I too configured AG in system boot.

One important think I noticed that ... can open an instrument list from
Kontakt library but after opening.... can’t load it from AG plugin side.
In this situation I manage to load from Server side.

Some times , some plugins Brouwer window ( instrument list )may be little
longer...in that situation , in AG plugin side can’t show that perfectly.

Thank you,
